### PR TITLE
TN-5045: Add exclude action flows flag for batch export and import

### PR DESCRIPTION
**Pull request by fisgeci**
#### Description

Please include a description of the change and why the change was made.

#### Checklist

- [x] I have self-reviewed this PR
- [x] I have tested the change and proved that it works in different scenarios
- [x] I have updated docs if needed

### CODE REVIEW

File: DOCUMENTATION.md
Comment (LaberionAjvazi) [inline DOCUMENTATION.md:322]: Should this be `If you want` (affirmative) ?

File: src/api/package-api.ts
Comment (LaberionAjvazi) [inline src/api/package-api.ts:25]: Can we skip this `if` and send the flag anyways?

File: src/content-cli-export.ts
Comment (LaberionAjvazi) [inline src/content-cli-export.ts:16]: Can we make sure to cast this to boolean or leave it as empty string to avoid evaluating to true for string `"false"`?

File: DOCUMENTATION.md
Comment (jetakasabaqi) [inline DOCUMENTATION.md:322]: nit: If you **want**

CDP service would have to send this new flag always, to fix the bug. Were they notified of the change?

Comment (fisgeci) [reply DOCUMENTATION.md:322]: I contacted with CDP team. After we merge this change. They will be able to upgrade the version of content-cli and adapt the changes on their end.

**Comment by fisgeci** [comment]
Labeling as do not merge till backend changes reach prod.

**Review by LaberionAjvazi — APPROVED** [review]

**Review by ashishkubaghel — APPROVED** [review]

**Pull request merged**
